src/js/components/header/LanguageDropdown.tsx

```diff
@@ -1,21 +1,61 @@
 import React, { FunctionComponent } from 'react';
 import { useDispatch } from 'react-redux';
 
-import { setLanguage } from '../../store/appState/actions';
+import { setLanguage } from 'js/store/appState/actions';
 
-const LanguageDropdown: FunctionComponent = () => {
+interface DropProps {
+  language: string;
+  alternativeLanguage: string;
+  selectedLanguage: string;
+}
+
+function valueToLang(abbrev: string): string {
+  let lang = '';
+
```

Comment:
we could do a lang map that could be easily extracted into some config later on: 
```
const langMap = {
  en: 'English',
  fr: 'French',
}
```
and then access `{langMap.hasOwnProperty(selectedLanguage) && langMap[selectedLanguage]}`
thoughts?

Reply:
Love it, hold that thought - I added a `Code Cleanup` [tag](https://github.com/wri/gfw-mapbuilder/issues/labels) that is perfect for this case. I will make a ticket for this and demand that we hoist this configuration elsewhere

Reply:
https://github.com/wri/gfw-mapbuilder/issues/768